src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/afcalc/AlleleFrequencyCalculator.java

```diff
@@ -75,7 +77,7 @@ public static AlleleFrequencyCalculator makeCalculator(final DragstrParams drags
         final double[] log10Likelihoods;
         if (g.hasLikelihoods()) {
             log10Likelihoods = g.getLikelihoods().getAsVector();
-        } else if ( g.isHomRef() || g.isNoCall()) {
+        } else if (g.isHomRef()) {
```

Comment:
So now we'll always throw an `IllegalStateException` here for no calls. Is that what you want?

Reply:
Traditionally GenomicsDB didn't output calls so ref blocks would also be ./., but I got anxious about genotypes with no call and no likelihoods.  Instead I just turned on genotype calling in GenomicsDB.